scripts/test.lua

```diff
@@ -7,15 +7,16 @@
    - cb_replace => Replace record content with a new table
 
    The key inside each function is to do a proper handling of the
-   return values. Each function must return 3 values:
+   return values. Each function must return 4 values:
```

Comment:
The function should still return 3 values (code, timestamp, record).

Reply:
@l2dy Oops! I was misunderstanding. 
I sent a patch to fix it. https://github.com/fluent/fluent-bit/pull/2117
Thank you.